src/uct/ib/ud/base/ud_iface.h

```diff
@@ -108,6 +108,7 @@ typedef struct uct_ud_iface_ops {
     void                      (*ep_free)(uct_ep_h ep);
     ucs_status_t              (*create_qp)(uct_ib_iface_t *iface, uct_ib_qp_attr_t *attr,
                                            struct ibv_qp **qp_p);
+    void                      (*destroy_qp)(uct_ib_iface_t *iface);
```

Comment:
better pass `ud_iface` as it is UD function. 
`uct_ud_verbs_iface_destroy_qp` will be 1-line then

Reply:
> better pass `ud_iface` as it is UD function.
> `uct_ud_verbs_iface_destroy_qp` will be 1-line then

done